interface/main/tabs/js/user_data_view_model.js

```diff
@@ -36,4 +36,11 @@ function userPrefs()
 {
     navigateTab(webroot_url+"/interface/super/edit_globals.php?mode=user","msc");
     activateTabByName("msc",true);
+}
+
+// this function is calling when clicking on "x unread messages" anchor on upper right corner
+// this opens the messages tab
+function unreadMessages() {
+    navigateTab(webroot_url+"/interface/main/messages/messages.php?form_active=1", "pat");
+    activateTabByName("pat", true);
 }
```

Comment:
Please add a new line to normalize this file.  It helps when we clean up this stuff as we find it.